src/veins/modules/mobility/traci/NetworkBoundary.h

```diff
@@ -0,0 +1,59 @@
+//
+// Copyright (C) 2018 Dominik S. Buse <buse@ccs-labs.org>
+//
+// Documentation for these modules is at http://veins.car2x.org/
+//
+// This program is free software; you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation; either version 2 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program; if not, write to the Free Software
+// Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+//
+
+#ifndef VEINS_MOBILITY_TRACI_NETWORKBOUNDARY_H_
+#define VEINS_MOBILITY_TRACI_NETWORKBOUNDARY_H_
+
+#include "veins/modules/mobility/traci/TraCICoord.h"
+#include "veins/base/utils/Coord.h"
+
+#include <list>
+
+namespace Veins {
+
+/*
+ * Helper class for converting SUMO coordinates to OMNeT++ Coordinates for a given network.
+ */
+class NetworkBoundary
```

Comment:
The class lacks documentation for its members, particullarly for the methods involving angles, without looking at the code it's unclear whether the angles are given in radians or degrees.